domains/shiv.json

```diff
@@ -1,9 +1,9 @@
-{
-  "owner": {
-    "username": "shivasharanappaBiradar",
-    "email": "youcanbethemaster@gmail.com"
-  },
-  "record": {
-    "CNAME": "railway-production-5fc6.up.railway.app"
-  }
-}
+"description": "Personal portfolio",
+    "repo": "https://shivasharanappabiradar2645.github.io/shivasharanappaBiradar2645/",
+    "owner": {
+        "username": "ShivasharanappaBiradar",
+        "email": "youcanbethemaster@gmail.com",
+    },
+    "record": {
+        "CNAME": "https://shivasharanappabiradar2645.github.io"
+    }
```

Comment:
You're JSON is broken, and the CNAME record should be a hostname without any protocols. Also the link you provided in your CNAME is returning a 404 site.